src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -2775,7 +2774,7 @@ export const handleSetGoLinkSuffix: ControllerHandler<
             },
           ),
           // TODO: fix error handling (https://linear.app/ogp/issue/FRM-901/improve-error-handling-when-calling-gogov-api)
-          () => new ApplicationError('Error occurred when claiming GoGov link'),
+          () => new GoGovError(),
```

Comment:
Non-issue. To note that this might be swallowing real errors such as invalid credentials/server down etc. Best to map over some expected errors (if GoGov team publishes them).

Reply:
Yep yep, planning to get those for the actual PR tackling [FRM-901](https://linear.app/ogp/issue/FRM-901)

Reply:
as mentioned by @mayying, they do have the errors here: https://guide.go.gov.sg/developer-guide/api-documentation#errors